.travis.yml

```diff
@@ -3,26 +3,26 @@ sudo: false
 matrix:
     include:
         - os: linux
-          python: pypy-5.4.1
+          python: pypy-5.6.0
         - os: linux
-          python: pypy3
+          python: pypy3.3-5.5-alpha
           env: BUILOUT_OPTIONS=sphinx:eggs=
         - os: linux
           python: 2.7
         - os: linux
-          python: 3.3
-        - os: linux
           python: 3.4
         - os: linux
           python: 3.5
+        - os: linux
+          python: 3.6
 install:
     - pip install -U pip
-    - pip install zc.buildout
-    - buildout $BUILOUT_OPTIONS versions:sphinx=1.4.9
+    - pip install -U setuptools zc.buildout
+    - buildout $BUILOUT_OPTIONS
 script:
     - if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then bin/coverage run bin/coverage-test -v1j99; fi
-    - if [[ $TRAVIS_PYTHON_VERSION == 'pypy' || $TRAVIS_PYTHON_VERSION == 'pypy3' ]]; then bin/test -v1j99; fi
-    - if [[ $TRAVIS_PYTHON_VERSION != 'pypy3' ]]; then pushd doc; make html; popd; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == pypy* ]]; then bin/test -v1j99; fi
+    - if [[ $TRAVIS_PYTHON_VERSION != pypy3* ]]; then pushd doc; make html; popd; fi
```

Comment:
Minor: there's an opportunity to use `make -C html doc` instead of the pushd/make/popd dance.